content/departments/product-engineering/engineering/cloud/delivery/managed/resume_process.md

```diff
@@ -0,0 +1,127 @@
+# Resuming a managed instance
+
+This page documents how to resume suspended [managed instances](./index.md). This is useful when Sales team would like to re-engage a customer again where their managed instance was suspended.
+
+Managed instances configuration is tracked in [deploy-sourcegraph-managed].
+
+For basic operations like accessing an instance for these steps, see [managed instances operations](operations.md).
+
+- [General setup](#general-setup)
+- [Sourcegraph resume](#sourcegraph-resume)
+  - [0) Sourcegraph resume setup](#0-sourcegraph-resume-setup)
+  - [1) Prepare terraform module](#1-prepare-terraform-module)
+  - [2) Make database writable](#2-make-database-writable)
+  - [3) Verify instance heath](#3-verify-instance-heath)
+  - [4) Commit your change](#4-commit-your-change)
+  - [5) Upgrade as needed](#5-upgrade-as-needed)
+
+## General setup
+
+Managed instances configuration is tracked in [deploy-sourcegraph-managed] - make sure you have the latest revision of this repository checked out. For basic operations like accessing an instance for these steps, see [managed instances operations](operations.md).
+
+First, ensure you have the prerequisites installed and up-to-date:
+
+- [Terraform CLI](https://www.terraform.io/)
+- [Deno CLI](https://deno.land/) (used for scripting upgrades)
+- [Comby CLI](https://comby.dev/) (used for rewriting configuration)
+- [jq](https://stedolan.github.io/jq/)
+
+Many of the following commands in this guide, as well as the commands [operations](./operations.md), use environment variables. Export the appropriate values for the upgrade so you don't lose track:
+
+```sh
+# name of customer deployment (should match folder)
+export CUSTOMER=<customer_or_instance_name>
+
+# the previous active deployment, either `red` or `black`
+export CURRENT_DEPLOYMENT=$(gcloud compute snapshots list --format json --sort-by '~creationTimestamp' --limit 1 | jq -r '.[0].name' | awk -F: '{ if ($1 ~ "-red-") {print "red"} else {print "black"} }')
```

Comment:
the one-liner roughly translates to `SELECT name FROM snapshots ORDER BY timestamp DESC LIMIT 1` 

this can ensure we get the latest snapshot just in case there's more than one.